### PR TITLE
use span for langkeywords and (type)paramref

### PR DESCRIPTION
**Pull request by superyyrrzz**
TASK 798457
@vwxyzh @ansyral @vicancy @hellosnow @qinezh @chenkennt

### CODE REVIEW

File: src/docfx.website.themes/default/styles/docfx.css [hunk 1]
```diff
@@ -89,6 +89,18 @@ span.parametername {
     font-style: italic;
 }
 
+span.paramref {
```
Comment (hellosnow) [inline src/docfx.website.themes/default/styles/docfx.css]: >span.paramref [](start = 0, length = 13)

merge to one line such as line 40?

**Comment by qinezh** [comment]
:shipit:

**Comment by ansyral** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**